tests/test_counting_hash.py

```diff
@@ -974,7 +985,9 @@ def test_get_ksize():
 
 def test_get_hashsizes():
     kh = khmer.CountingHash(22, 100, 4)
-    assert kh.hashsizes() == [97L, 89L, 83L, 79L], kh.hashsizes()
+    expected = utils.longify([97, 89, 83, 79])  # Py3 hack.
```

Comment:
It's a work-around for not having long literals, and needing them in py2. Is this an acceptable solution?